### PR TITLE
Add command completion to Ramalama

### PR DESCRIPTION
**Pull request by rhatdan**
I had to move some things around to process in ramalama.py to make this work.

To try this out, you will need to install it and recreate your shell.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ jobs:
         shell: bash
         run: |
            sudo apt-get update
-           sudo apt-get install bats bash
```
Comment (ericcurtin) [inline .github/workflows/ci.yml:12]: Do we need this installed twice?

argcomplete

Comment (rhatdan) [reply .github/workflows/ci.yml:12]: Nope, I think the issue I am seeing is because argcomplete is not in the Container.

File: Makefile [hunk 2]
```diff
@@ -40,6 +40,18 @@ install-requirements:
 	pip install "huggingface_hub[cli]==0.25.1"
 	pip install "omlmd==0.1.4"
 
+.PHONY:
+install-completions:
+	install ${SELINUXOPT} -d -m 755 $(DESTDIR)${SHAREDIR}/bash-completion/completions
+	register-python-argcomplete --shell bash ramalamay > $(DESTDIR)${SHAREDIR}/bash-completion/completions/ramalama
```
Comment (ericcurtin) [inline Makefile:46]: Typo s/ramalamay/ramalama/g

Can catch in a follow on PR if needs be

File: rpm/python-ramalama.spec [hunk 3]
```diff
@@ -57,13 +58,19 @@ Provides: %{pypi_name} = %{version}-%{release}
...
+%{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} install-completions
 
 %files -n python%{python3_pkgversion}-%{pypi_name} -f %{pyproject_files}
 %license LICENSE
 %doc README.md
 %dir %{_datadir}/%{pypi_name}
 %{_datadir}/%{pypi_name}/shortnames.conf
 %{_mandir}/man1/ramalama*.1*
+%{_datadir}/bash-completion/completions/%{pypi_name}
+%{_datadir}/fish/vendor_completions.d//%{pypi_name}.fish
```
Comment (ericcurtin) [inline rpm/python-ramalama.spec:70]: Double // , very minor nit

**Pull request merged**